Fixes #560 and provides an initial implementation for assigning decorators to tags.

deepkn opened the pull request:
This patch enables decorations for user specified tags - starting with handling only a single decoration - a 'prefix' to be added to the page name. Prefix is handled in the top bar title, page navigator, wiki-links appearing within a page as well as page autocomplete suggestions.

deepkn commented:
Hey Zef, resolved conflicts with main and squashed the latest into a single commit; I might have accidentally closed the original PR - sorry about that.

zefhemel commented:
I merged this, but I will still make some changes. Also in the way the decorations are specified. I'll let you know when you can try it.

zefhemel commented:
Ok, I worked this a little bit: https://silverbullet.md/Page%20Decorations

deepkn commented:
Thanks Zef. Just moved to latest edge - excited to use the new features.

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

On Sun, Jul 14, 2024 at 3:08 PM, Zef Hemel ***@***.***(mailto:On Sun, Jul 14, 2024 at 3:08 PM, Zef Hemel <<a href=)> wrote:

> Ok, I reworked this a little bit: https://silverbullet.md/Page%20Decorations
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/silverbulletmd/silverbullet/pull/940#issuecomment-2227282308), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AAIUDVKNTOIDMRE6BBSB6B3ZMJBJBAVCNFSM6AAAAABK2DJUCKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMRXGI4DEMZQHA).
> You are receiving this because you authored the thread.Message ID: ***@***.***>
